ProjectTemplates/templates/ComponentsWebAssembly-CSharp/Server/Controllers/WeatherForecastController.cs

```diff
@@ -38,7 +38,7 @@ public class WeatherForecastController : ControllerBase
         private readonly ILogger<WeatherForecastController> _logger;
 
         // The Web API will only accept tokens 1) for users, and 2) having the access_as_user scope for this API
-        static readonly string[] scopeRequiredByApi = new string[] { "access_as_user" };
+        static readonly string[] scopeRequiredByApi = new string[] { "api-scope" };
 
```

Comment:
i don't think this is used anymore

Reply:
nevermind. i see it. in code flow it didn't show that line because it wasn't changed. never mind.